source/ddbc/drivers/odbcddbc.d

```diff
@@ -558,18 +558,63 @@ version (USE_ODBC)
         override void setAutoCommit(bool autoCommit)
         {
             checkClosed();
-            if (this.autocommit != autocommit)
+            if (this.autocommit != autoCommit)
             {
                 lock();
                 scope (exit)
                     unlock();
 
                 uint ac = autoCommit ? SQL_AUTOCOMMIT_ON : SQL_AUTOCOMMIT_OFF;
 
-                checkdbc!SQLSetConnectAttr(conn, SQL_ATTR_AUTOCOMMIT, &ac, SQL_IS_UINTEGER);
+                writeln("Setting connection ac=", ac);
```

Comment:
please don't use writeln. It's fine to use `std.logger` but please see other parts of code that do import in a way that is backwards compatible:

```
static if (__traits(compiles, (){ import std.logger; } )) {
    import std.logger;
} else {
    import std.experimental.logger;
}
```

Reply:
Ah sorry about that, this was just debugging output while I was figuring things out. I can delete it entirely.